src/cmds/user/index.js

```diff
@@ -183,6 +184,22 @@ class UserCmd {
             });
         });
     }
+
+    async batchDeleteUsers() {
+        let users;
+
+        do {
+            const results = await userService.listUsers();
+            users = results.Users.map(users => users.Username);
+            logger.info(`Number of users found ${users.length}`);
+
+            await Promise.map(users, async username => {
+                await userService.deleteUsers(username)
+            }, {concurrency: 3});
+
```

Comment:
Can we do a final prompt here so that the user doesn't accidentally delete everything?